project.yaml

```diff
@@ -10,7 +10,7 @@ network:
   genesisHash: '0x97da7ede98d7bad4e36b4d734b6055425a3be036da2a332ea5a7037656427a21'
 dataSources:
   - kind: substrate/Runtime
-    startBlock: 343031
+    startBlock: 401
```

Comment:
@hobbydev71 Just make sure we go all the way down to 1 for the actual deployment.